code/game/objects/items/pet_carrier.dm

```diff
@@ -89,16 +92,19 @@
 	if(target.mob_size > max_occupant_weight)
 		if(ishuman(target))
```

Comment:
This feature could honestly also be expanded to monkeys, all it has be done is change a couple is_type checks, and adjust the to_chat messages at line 99.
The `var/mob/living/carbon/human/H = target` is unnecessary, resting is a living var and the iscatperson define will halt if the mob is not human.